src/runtime/slot-consumer.ts

```diff
@@ -184,7 +184,9 @@ export class SlotConsumer {
     });
   }
 
-  isSameContainer(container, contextContainer) { return container === contextContainer; }
+  isSameContainer(container, contextContainer) {
+    return (!container === !contextContainer) && (container === contextContainer);
```

Comment:
This is an icky area of JS, but I think just changing `===` to `==` one line 187 is easier

Reply:
Sigh, if you can't use `==` then I think you should make a method isNullOrUndefined (or isNullish, or something).

Yes, IMO
```
return !container && !contextContainer  || (container === contextContainer);
```
is better.